src/renderer/coremods/settings/pages/QuickCSS.css

```diff
@@ -72,3 +75,19 @@
   width: 100%;
   height: 100%;
 }
+
+/* SB: If you remove this section of the code, the Discord Title bar will show up which is
+annoying since it's black and well with Light Theme it don't make sense bud. */
+[class^="popout-"]:has(#replugged-quickcss-wrapper) [class^="content-"] {
```

Comment:
Remove. All Discord's popouts have the titlebar dark themed, keep it as it is. And the "Stay On Top" button shouldn't be on the titlebar neither.